src/main/java/application/onlinebookstore/service/impl/OrdersServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package application.onlinebookstore.service.impl;
+
+import application.onlinebookstore.dto.cartitem.CartItemDto;
+import application.onlinebookstore.dto.orderitem.CreateOrderItemDto;
+import application.onlinebookstore.dto.orderitem.OrderItemDto;
+import application.onlinebookstore.dto.orders.CreateOrderDto;
+import application.onlinebookstore.dto.orders.OrderDto;
+import application.onlinebookstore.dto.shoppingcart.ShoppingCartDto;
+import application.onlinebookstore.exception.EntityNotFoundException;
+import application.onlinebookstore.mapper.OrderItemMapper;
+import application.onlinebookstore.mapper.OrderMapper;
+import application.onlinebookstore.model.Orders;
+import application.onlinebookstore.model.User;
+import application.onlinebookstore.repository.orders.OrdersRepository;
+import application.onlinebookstore.service.OrderItemService;
+import application.onlinebookstore.service.OrdersService;
+import application.onlinebookstore.service.ShoppingCartService;
+import application.onlinebookstore.service.UserService;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class OrdersServiceImpl implements OrdersService {
+    private final OrdersRepository ordersRepository;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final OrderItemService orderItemService;
+    private final OrderItemMapper orderItemMapper;
+    private final OrderMapper orderMapper;
+
+    @Override
+    public List<OrderDto> getOrders(Long userId) {
+        return ordersRepository.findAllByUserId(userId).stream()
+                .map(orderMapper::toDto)
+                .toList();
+    }
+
+    @Override

```

Comment:
```suggestion
    @Override
    @Transactional
```
Think carefully which methods need to be transactional